clifford/test/test_multivector_inverse.py

```diff
@@ -6,16 +6,37 @@
 
 class TestClosedForm:
 
+    @pytest.mark.parametrize('p, q, r', [
+        (p, p_add_q - p, r)
+        for p_add_q in [1, 2, 3, 4, 5]
+        for p in range(p_add_q + 1)
+        for r in range(2)
+        if r + p_add_q <= 5  # beyond this the hitzer_inverse is not supported
+    ])
+    def test_hitzer_inverse(self, p, q, r):
+        Ntests = 100
+        layout, blades = cf.Cl(p, q, r)
+        for i in range(Ntests):
+            mv = layout.randomMV()
+            mv_inv = mv.hitzer_inverse()
+            np.testing.assert_allclose((mv * mv_inv).value,
+                                       layout.scalar.value,
+                                       rtol=1E-5,
+                                       atol=1E-6)
+
+    @pytest.mark.parametrize('r', range(2))
     @pytest.mark.parametrize('p, q', [
         (p, total_dims - p)
```

Comment:
https://github.com/pygae/clifford/pull/373/files#r549352578 still applies. At a guess:
```suggestion
        pytest.param(p, total_dims - p, marks=[pytest.mark.slow] if total_dims >= 6 else [])
```